src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx

```diff
@@ -0,0 +1,128 @@
+
+You need to install the New Relic errors inbox for Jira to keep your data in sync between New Relic and Atlassian. Keep in mind that you must have the Jira site administrator permission to install marketplace applications. Go to [Atlassian Marketplace](https://marketplace.atlassian.com/apps/1227457/new-relic-errors-inbox-for-jira?hosting=cloud&tab=overview) and click the **Install** button.
+
+<Callout variant="tip">
+  If you don't have the site admin permission, you can request it from your administrator. We recommend that you follow up with your Jira site administrator to complete the application install process.
+</Callout>
+
+</Step>
+<Step>
+
+## Create a New Relic API key [#create-new-relic-key]
+
+After the installation of the New Relic Atlassian Marketplace application, you need to create a [New Relic API key](/docs/apis/intro-apis/new-relic-api-keys/). The Jira site administrator and the New Relic administrator will decide on the exposed accounts to Jira. The users that have the correct New Relic account access, could generate a New Relic API key. The New Relic API keys have access to the same accounts as the user who generated them.
+
+Create and manage your API keys from the [New Relic API keys UI](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher) page or log in to the [Atlassian API tokens](https://id.atlassian.com/manage-profile/security/api-tokens) page and click **Create API token**.
+
+</Step>
+
+<Step>
+## Apply a New Relic API key [#apply-new-relic-key]
+
+Once you've created an API key, you can apply that key to the first page of the New Relic connect application. 
+
+Paste your New Relic API token and click **Connect**.
+
+    <img
+    width="50%;"
+    title="Apply a New Relic API key"
+    alt="Apply a New Relic API key"
+    src={opentelemetryAtlassianNRapiToken}
+    />
+
+
+</Step>
+<Step>
+## Select your New Relic services [#select-nr-services]
+
+Once you've created an API key in the New Relic Atlassian Marketplace application, you'll see a list of all entities associated with the accounts on that API key. Open the Jira incident management page and select all the entities that you want to expose in Jira.
+
+    <img
+    title="Jira incident management page"
+    alt="Jira incident management page"
+    src={opentelemetryJiraIncidentManagement}
+    />
+
+
+</Step>
+<Step>
+## See New Relic issues and incidents in Atlassian Jira [#see-issues-incidents]
+
+After you've selected your New Relic entities, the application starts syncing data from New Relic into Atlassian Jira. Open the incidents tab to see a list of detected incidents.
+
+
+</Step>
+
+</Steps>
+
+## Enable the incidents tab [#enable-incidents-tab]
+
+You may need to link to the Atlassian documentation. It walks a Jira site admin through enabling the Incidents tab and assigning it to a workspace. There is a chance they are going to enable by default and then, you won't need this section.
```

Comment:
I notice that Incidents and incident tab and the like have some inconsistent capitalization that will need to be addressed

Reply:
There is > There's

Reply:
"a chance they are going to enable by default" reads very awkwardly to me